### PR TITLE
Autosave

### PR DESCRIPTION
**Pull request by magsouza**

### CODE REVIEW

File: vitaes-webapp/src/Builder/index.js [hunk 1]
```diff
@@ -36,15 +36,19 @@ class Builder extends Component {
         'skill',
       ],
       params: {},
+      saving: false,
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/Builder/index.js]:
```suggestion
      autoSave: false,
```

File: vitaes-webapp/src/Builder/index.js [hunk 2]
```diff
@@ -151,6 +155,15 @@ class Builder extends Component {
     }
   }
 
+  autoSave() {
+    setInterval(() => {
+      if (this.state.saving) {
+        this.saveOnAccount()
+        toast.success(translate('saved'))
+      }
+    }, 180000)
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/Builder/index.js]: tune it down to 2 min

File: vitaes-webapp/src/Builder/index.js [hunk 3]
```diff
@@ -225,6 +238,14 @@ class Builder extends Component {
         <Card.Body>
           <h2>Curriculum Vitae:</h2>
           <br />
+          <Form.Check
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/Builder/index.js]: Can we have pictures 😮 ?

File: vitaes-webapp/src/Builder/index.js (hunk 2 above)
Comment (rafaellsma) [reply vitaes-webapp/src/Builder/index.js]: Put into a const too. :grin:

**Pull request closed without merging**